dist/index.js

```diff
@@ -15946,7 +15946,8 @@ const context = {
 	}),
 	ALIAS_DOMAINS: parser.getInput({
 		key: 'ALIAS_DOMAINS',
-		type: 'array'
+		type: 'array',
+		disableable: true
```

Comment:
not sure why this change happened, this is a result of running `npm run build` in the repo.